dotcom-rendering/src/lib/decidePalette.ts

```diff
@@ -1150,40 +1150,73 @@ const fillShareIconGrayBackground = (format: ArticleFormat): string => {
 };
 
 const fillBlockquoteIcon = (format: ArticleFormat): string => {
-	if (format.design === ArticleDesign.Analysis) {
-		switch (format.theme) {
-			case Pillar.News:
-				return news[300];
-			default:
-				return pillarPalette[format.theme].main;
+	switch (format.design) {
+		case ArticleDesign.LiveBlog:
+		case ArticleDesign.DeadBlog: {
+			switch (format.theme) {
+				case Pillar.News:
+					return news[400];
+				case Pillar.Opinion:
+					return opinion[400];
+				case Pillar.Sport:
+					return sport[400];
+				case Pillar.Culture:
+					return culture[400];
+				case Pillar.Lifestyle:
+					return lifestyle[400];
+				case ArticleSpecial.SpecialReport:
+					return specialReport[400];
+				case ArticleSpecial.SpecialReportAlt:
+					return news[400];
+				case ArticleSpecial.Labs:
+					return labs[400];
+				default:
+					return neutral[86];
```

Comment:
It never reaches the default case. I'm only adding it because TS is complaining. Let me know if there's a better way!

Reply:
What’s the error?

Reply:
If I remove it IntelliJ says:

![image](https://github.com/guardian/dotcom-rendering/assets/19683595/fd95c395-882c-40ee-b02e-3ca19bbfc052)

Haven't ran `yarn tsc` to verify to be fair.

Reply:
Oh `yarn tsc` is happy. Shall I remove these default cases?

Reply:
Removed